ova/generate_ova.sh

```diff
@@ -25,28 +25,37 @@ HAVE_VERSION=false
 HAVE_OPENDISTRO_VERSION=false
 HAVE_ELK_VERSION=false
 
+# Get the currents production branches
+BRANCHDOC=$(curl -s https://documentation.wazuh.com/current/index.html | grep -o "m/[0-9\.]\+/i" | head -n1)
+BRANCHDOC="${BRANCHDOC:2:3}"
+BRANCH=${BRANCHDOC}
+export BRANCHDOC
+export BRANCH
+
 WAZUH_VERSION=""
 OPENDISTRO_VERSION=""
-BRANCH="master"
 ELK_VERSION=""
-PACKAGES_REPOSITORY=""
+PACKAGES_REPOSITORY="prod"
 CHECKSUM="no"
 UI_REVISION="1"
 
 help () {
 
     echo
-    echo "Usage: $0 [OPTIONS]"
-    echo "  -v, --version          [Required] Version of wazuh to install on VM."
-    echo "  -o, --opendistro       [Required] Version of Open Distro for Elasticsearch."
-    echo "  -f, --filebeat         [Required] Filebeat's version."
-    echo "  -r, --repository       [Required] Select the software repository [prod/dev]."
-    echo "  -b, --branch           [Optional] Branch/tag of the Wazuh repository."
-    echo "  -s, --store <path>     [Optional] Set the destination absolute path of package."
-    echo "  -c, --checksum <path>  [Optional] Generate checksum."
-    echo "  -u, --ui-revision      [Optional] Revision of the UI package. By default, 1."
-    echo "  -h, --help             [  Util  ] Show this help."
+    echo "General usage: $0 [OPTIONS]"
+    echo "  -w,    --wazuh            [Required] Version of wazuh to install on VM."
+    echo "  -o,    --opendistro       [Required] Version of Open Distro for Elasticsearch."
+    echo "  -f,    --filebeat         [Required] Filebeat's version."
```

Comment:
It's not the filebeat version. Not only. It's the elasticsearch/kibana/filebeat version. 
I would say:
`[Required] Version of Opendistro filebeat/elasticsearch/kibana components.`

Reply:
Fixed in https://github.com/wazuh/wazuh-packages/pull/746/commits/ee4a49213431a56ed579126d6fe9179cfb02dcf4